jupyterlab_gcedetails/src/data.ts

```diff
@@ -391,3 +337,8 @@ export const STYLES = stylesheet({
     boxShadow: 'inset 0 -1px 0 0 var(--jp-border-color0)',
   },
 });
+
+export const TEXT_STYLE = {
+  fontFamily: BASE_FONT.fontFamily as string, //'var(--jp-ui-font-family)'
```

Comment:
Are the explicit type annotations necessary here?

Reply:
Because I'm passing this const to Material ui components directly as props, it starts complaining if the type is not explicitly set to string (as `BASE_FONT.fontFamily` has type `string | string[]`). I can probably figure out a workaround though if this is a concern? :)

Reply:
Nah this good.